src/common/utils/url.js

```diff
@@ -10,15 +10,13 @@ import { isEmpty } from 'lodash';
  * @returns Whether a url is a kitsu url.
  */
 export function isKitsuUrl(url) {
-  if (isEmpty(url)) return false;
-
   // Parse it
-  const parsed = parseURL(url);
+  const parsed = parseURL((url || ''));
```

Comment:
You can do the following instead of `||` everywhere:
```js
export function isKitsuUrl(url = '') {
```

Reply:
Ah thanks, not used to JS still, i'll quickly commit the changes.

Reply:
A quick question, does doing the change above protect the url from being `null` or `undefined`?

I tried this in REPL
```
function isUrl(url = 'default') {
  console.log(url);
}

isUrl();
isUrl('abc');
isUrl(null);
```
and it outputted
```
default
abc
null
```

Reply:
Oh I guess the current `|| ''` is more ideal in this case.